client/src/leap/soledad/client/encdecpool.py

```diff
@@ -507,15 +446,12 @@ def insert_encrypted_received_doc(
         :param idx: The index of this document in the current sync process.
         :type idx: int
```

Comment:
Maybe we can update the docstring. As far as I understand this method doesn't insert the document for latter decryption.